applications/GeoMechanicsApplication/tests/cpp_tests/custom_processes/test_apply_initial_uniform_stress_field.cpp

```diff
@@ -0,0 +1,133 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_constitutive/incremental_linear_elastic_law.h"
+#include "custom_constitutive/plane_strain.h"
+#include "custom_constitutive/three_dimensional.h"
+#include "custom_processes/apply_initial_uniform_stress_field.h"
+#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
+#include "tests/cpp_tests/test_utilities/model_setup_utilities.h"
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(ApplyInitialUniformStressFieldProcessAppliesStressesToPlaneStrainElementsInModelParts,
```

Comment:
The process expects exactly one model part if I'm not mistaken:
```suggestion
KRATOS_TEST_CASE_IN_SUITE(ApplyInitialUniformStressFieldProcessAppliesStressesToPlaneStrainElementsInModelPart,
```

Reply:
Done